### PR TITLE
feat: add notebook access event

### LINKED ISSUE #3217: add a metric for notebook page views
every time a notebook is viewed.. send an event. we're going to add it as a metric to track on the notebooks usage notebook in tools

### PR DESCRIPTION
**Pull request by subirjolly**
Closes #3217

### CODE REVIEW

File: src/flows/components/FlowPage.tsx [hunk 1]
```diff
@@ -34,6 +36,12 @@ const FlowFromRoute = () => {
     change(id)
   }, [id, change])
 
+  useEffect(() => {
+    if (currentID !== null) {
+      event(`Notebook Accessed. NotebookID: ${currentID}`)
```
Comment (drdelambre) [inline src/flows/components/FlowPage.tsx:41]: Remove the notebook I'd stuff after the period or else we won't be able to group these. If you want to keep that granularity, move it into the tags parameter,  evt(name, { tag: tagval })

Comment (subirjolly) [reply src/flows/components/FlowPage.tsx:41]: makes sense.

**Review by drdelambre — APPROVED** [review]
I don't understand the ramifications of putting this in a useEffect vs a route matcher, but let's find out together!

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 59 | :red_circle:
accessibility | 97 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=3282)

**Pull request merged**